packages/fluent-theme/package.json

```diff
@@ -26,6 +26,7 @@
     "@uifabric/build": "^7.0.0"
   },
   "dependencies": {
+    "@fluentui/theme": "^0.1.0",
```

Comment:
it seems like the directory should just be `theme` if we call it `@fluentui/theme`.